api/middleware/requests.ts

```diff
@@ -0,0 +1,5 @@
+import { Request } from 'express';

```

Comment:
Dlaczego dalismy to do middleware?
Stwórzmy onwy folder nazwijmy go types.ts i tam to przeniesmy